### PR TITLE
Vanguard Overhaul

### PR DESCRIPTION
**Pull request by TyroneMF**
- Added Guard branch
- Added Sand/Green branches
- Added more vehicles

- Removed generic "contractors" (replaced with Sand/Green branch)
- Other minor changes to equipment

### CODE REVIEW

**Comment by TyroneMF** [comment]
@Kresky look this over before it gets merged please.

**Comment by Kresky** [comment]
Remove the `TACU_` prefix from the UI images.
Can be done in batch with a PowerShell script:
`get-childitem *.jpg | foreach { rename-item $_ $_.Name.Replace("TACU_", "") }`
(I think that's how the script should be. Backup images just in case before you try probably).

And don't forget to update editorPreview's, thankfully "find and replace" makes that part easy.

File: addons/vanguard/CfgVehicles_Contractors_Green.hpp [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    respawnWeapons[] = {"arifle_Mk20_ACO_F", "hgun_Pistol_heavy_01_green_F", "Throw", "Put"};
+    magazines[] = {mag_7("30Rnd_556x45_Stanag"), mag_2("11Rnd_45ACP_Mag"), mag_2("HandGrenade"), "SmokeShell"};
+    respawnMagazines[] = {mag_7("30Rnd_556x45_Stanag"), mag_2("11Rnd_45ACP_Mag"), mag_2("HandGrenade"), "SmokeShell"};
+    headgearList[] = {
+        "H_HelmetSpecB", 1
+    };
+    TACU_allowedFacewear[] = {
+        "CUP_FR_NeckScarf", 1
+    };
+    editorSubcategory = "TACU_Vanguard_EdSubCat_Contractors1";
```
Comment (Kresky) [inline addons/vanguard/CfgVehicles_Contractors_Green.hpp]:
```suggestion
    editorSubcategory = "TACU_Vanguard_EdSubCat_Contractors_Green";
```

File: addons/vanguard/CfgVehicles_Contractors_Guards.hpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    respawnWeapons[] = {"SMG_03C_black", "CUP_hgun_M9", "Throw", "Put"};
+    magazines[] = {mag_3("50Rnd_570x28_SMG_03"), mag_2("CUP_15Rnd_9x19_M9"), mag_2("HandGrenade"), "SmokeShell"};
+    respawnMagazines[] = {mag_3("50Rnd_570x28_SMG_03"), mag_2("CUP_15Rnd_9x19_M9"), mag_2("HandGrenade"), "SmokeShell"};
+    headgearList[] = {
+        "None", 1
+    };
+    TACU_allowedFacewear[] = {
+        "G_Aviator", 1
+    };
+    editorSubcategory = "TACU_Vanguard_EdSubCat_Contractors3";
```
Comment (Kresky) [inline addons/vanguard/CfgVehicles_Contractors_Guards.hpp]:
```suggestion
    editorSubcategory = "TACU_Vanguard_EdSubCat_Guards";
```

File: addons/vanguard/CfgVehicles_Contractors_Sand.hpp [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    respawnWeapons[] = {"TACU_Vanguard_W_F2000_Sand", "hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    magazines[] = {mag_7("30Rnd_556x45_Stanag"), mag_2("11Rnd_45ACP_Mag"), mag_2("HandGrenade"), "SmokeShell"};
+    respawnMagazines[] = {mag_7("30Rnd_556x45_Stanag"), mag_2("11Rnd_45ACP_Mag"), mag_2("HandGrenade"), "SmokeShell"};
+    headgearList[] = {
+        "H_HelmetSpecB_sand", 1
+    };
+    TACU_allowedFacewear[] = {
+        "CUP_FR_NeckScarf2", 1
+    };
+    editorSubcategory = "TACU_Vanguard_EdSubCat_Contractors2";
```
Comment (Kresky) [inline addons/vanguard/CfgVehicles_Contractors_Sand.hpp]:
```suggestion
    editorSubcategory = "TACU_Vanguard_EdSubCat_Contractors_Sand";
```

**Pull request merged**